.github/workflows/update_sdk.yml

```diff
@@ -0,0 +1,132 @@
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.repository.default_branch }}
+        path: win32more
+
+    - name: Find latest version
+      run: |
+        $packageName = "${{ matrix.sdk.name }}".ToLower()
+        $uri = "https://api.nuget.org/v3-flatcontainer/$packageName/index.json"
+        $response = Invoke-RestMethod -Uri $uri
+        if ($${{ matrix.sdk.allow_prerelease}}) {
+          $versions = $response.versions
+        } else {
+          $versions = $response.versions | Where-Object { $_ -notmatch "-" }
+        }
+        if ($versions.Count -eq 0) {
+            throw "No versions found."
+        }
+        $latestVersion = $versions[-1]
+        $latestVersion = $latestVersion -replace '-preview$', ''
+        echo "latestVersion=$latestVersion" >> $env:GITHUB_ENV
+
+    - name: Verify if there is an update
+      id: is-there-update
+      run: |
+        $jsonPath = "win32generator\resources\metadata\versions.json"
+        $jsonContent = Get-Content -Path $jsonPath -Raw | ConvertFrom-Json
+        $version = $jsonContent."${{ matrix.sdk.name }}"
+        if ([System.Version]$env:latestVersion -le [System.Version]$version) {
+          Write-Output "There isn't any new version of ${{ matrix.sdk.name }}."
+          echo "isSuccess=false" >> $env:GITHUB_OUTPUT
+        } else {
+          echo "isSuccess=true" >> $env:GITHUB_OUTPUT
+        }
+      working-directory: win32more
+
+    - name: Checkout winmd-printer code
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      uses: actions/checkout@v4
+      with:
+        repository: ynkdir/winmd-printer
+        path: winmd-printer
+
+    - name: Setup .NET
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.x
+
+    - name: Setup Python
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.13"
+
+    - name: Setup 7-Zip
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: choco install 7zip --yes
+
+    - name: Run ${{ matrix.sdk.script_file_name }}
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: ${{ matrix.sdk.script_file_name }} $env:latestVersion
+      working-directory: winmd-printer
+
+    - name: Compress ${{ matrix.sdk.json_file_name }} with 7-Zip
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: |
+        Remove-Item win32more\win32generator\resources\metadata\${{ matrix.sdk.json_file_name }}.xz
+        7z a -txz win32more\win32generator\resources\metadata\${{ matrix.sdk.json_file_name }}.xz winmd-printer\${{ matrix.sdk.json_file_name }}
+
+    - name: Run win32generator
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: python -m win32generator
+      working-directory: win32more
+
```

Comment:
When updating WindowsAppSDK and WebView2, we need to run this script to get new runtime dll.
scripts/update_appsdk.py [version]
scripts/update_webview2.py [version]

Reply:
Fixed in https://github.com/ynkdir/py-win32more/pull/66/commits/10f2a20800ebf5318207e0ea17353176d3e35459 and https://github.com/ynkdir/py-win32more/pull/66/commits/bed5634d6833af9d9d2ac7fcc52f41977fae8404